tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -87,6 +87,7 @@ namespace Internal {
 namespace RDF {
 class GraphCreatorHelper;
 void ChangeEmptyEntryRange(const ROOT::RDF::RNode &node, std::pair<ULong64_t, ULong64_t> &&newRange);
+void ChangeEntryRange(const RNode &node, const ULong64_t start, const ULong64_t end);
```

Comment:
I would name this function (and the related one in RLoopManager) `ChangeBeginAndEndEntries` to be more specific of the different use case involved here. At some point we will need to decide how to provide a more coherent API for all the similar use cases